modules/oe_content_call_tenders/oe_content_call_tenders.post_update.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Call for tenders Content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add maxlegnth to oe_content_short_title, oe_summary, oe_teaser, title.
+ */
+function oe_content_call_tenders_post_update_00001() {
+  $file_storage = new FileStorage(drupal_get_path('module', 'oe_content_call_tenders') . '/config/post_updates/00001_add_maxlength');
```

Comment:
Maxlength module has to be enabled prior to updating form.

Reply:
no need, its dependency in oe_content.info.yml